packages/frontend/src/hooks/useCrabPosition/useCrabPosition.ts

```diff
@@ -118,10 +118,10 @@ export const useCrabPositionV2 = (user: string) => {
   const [minPnlUsd, setMinPnlUsd] = useState(BIG_ZERO)
   const [minPnL, setMinPnL] = useState(BIG_ZERO)
 
-  const { depositedEth, usdAmount: depositedUsd } = useAppMemo(() => {
-    if (txHistoryLoading || !txHistoryData) return { depositedEth: BIG_ZERO, usdAmount: BIG_ZERO }
+  const { depositedEth, usdAmount: depositedUsd, withdrawnEth, withdrawnUsdAmount,depositedValueEth,depositedValueUsd } = useAppMemo(() => {
```

Comment:
@KMKoushik can you confirm that adding these consts would not break any other part of the code?

Reply:
@DemolaJames Let's sync on this. I didn't understand the issue or fix completely!